foundation-week-4/t3q1.py

```diff
@@ -0,0 +1,26 @@
+from unittest import TestCase, main
+from t2q1 import authenticate, withdrawal
+
+class HomeworkTests(TestCase):
```

Comment:
⚠️ (-2) Great work on the unit tests. However, that would perfect if you could add the edge cases where user tries to input more than 3 times incorrect password.